ui.apps/src/main/content/jcr_root/apps/etoolbox-link-inspector/clientlibs/link-inspector-ui/js/console-ui.filter.js

```diff
@@ -122,7 +138,9 @@
     $(document).ready(function () {
        let searchParams = new URL(document.location).searchParams;
        if (searchParams != null && searchParams.get('type') != null || searchParams.get('substring') != null) {
-         $('#elc-filter-options').attr('variant', 'primary');
+         $('#elc-filter-options').addClass('elc-filter-active');
```

Comment:
Consider `$('#elc-filter-options').toggleClass('elc-filter-active', searchParams && (searchParams.get('type') || searchParams.get('substring')))`

You need to care that "type" and "substring" are  _truthy_ , don't you? Not exactly `null`. They might also be `undefined`, or `''`. Or am I wrong?